cmd/prombench/main.go

```diff
@@ -19,35 +19,35 @@ func main() {
 	k8sGKE := app.Command("gke", `Google container engine provider - https://cloud.google.com/kubernetes-engine/.Requires`).
 		Action(g.NewGKEClient).
 		Action(g.ConfigParse)
-	k8sGKE.Flag("config", "yaml GKE config file used to create or delete the k8s cluster and nodes").
-		Default("../../config/cluster.yaml").
-		PlaceHolder("cluster.yaml").
+	k8sGKE.Flag("config", "yaml GKE config file used to scale_up or scale_down the prow cluster").
+		PlaceHolder("node-pool.yaml").
 		Short('c').
+		Default("../../config/node-pool.yaml").
 		ExistingFileVar(&g.ClusterConfigFile)
 	k8sGKE.Flag("auth", "json authentication file for the project - https://cloud.google.com/iam/docs/creating-managing-service-account-keys. If not set the tool will use the GOOGLE_APPLICATION_CREDENTIALS env variable (export GOOGLE_APPLICATION_CREDENTIALS=key.json)").
 		PlaceHolder("key.json").
 		Short('a').
 		ExistingFileVar(&g.AuthFile)
+	k8sGKE.Flag("vars", "When provided it will substitute the token holders in the resources file. Follows the standard golang template formating - {{ .hashStable }}.").
+		Short('v').
+		StringMapVar(&g.ResourceVars)
 
-	k8sGKECluster := k8sGKE.Command("cluster", "Create or delete k8s clusters")
-	k8sGKECluster.Command("create", "gke cluster create -a key.json  -c ../../config/cluster.yaml").
-		Action(g.ClusterCreate)
-	k8sGKECluster.Command("delete", "gke cluster delete -a key.json  -c ../../config/cluster.yaml").
-		Action(g.ClusterDelete)
+	k8sGKECluster := k8sGKE.Command("cluster", "Scale up or down a k8s clusters by creating node-pools")
+	k8sGKECluster.Command("scaleUp", "gke cluster scaleUp -a key.json  -c ../../config/node-pool.yaml").
+		Action(g.NodePoolCreate)
+	k8sGKECluster.Command("scaleDown", "gke cluster scaleDown -a key.json  -c ../../config/node-pool.yaml").
+		Action(g.NodePoolDelete)
 
```

Comment:
can we  keep as there is a chance to need all of them at some point.

```
cluster create
cluster delete
nodePool create
nodePool delete
```

Reply:
[bb340ed](https://github.com/prometheus/prombench/pull/25/commits/bb340ede8db305abfa0256ac1052fd406ead00b6) adds these